nautobot/docs/release-notes/version-2.0.md

```diff
@@ -18,6 +18,26 @@ Added Site Model Fields to Location. Location Model now has `asn`, `comments`, `
 
 #### Collapse Region and Site Models into Location ([#2517](https://github.com/nautobot/nautobot/issues/2517))
 
+##### Initial Data Migration
+
+`Site` and `Region` models are removed in v2.0. As a result, the existing `Site` and `Region` data will be moved to `LocationTypes` and `Locations`. Here is what to expect:
+
+1. If you do not have any `Site` and `Region` instances in your existing database, running this data migration will do nothing.
+2. If you only have `Region` instances in your existing database, a `Region` LocationType will be created and for each legacy `Region` instance, a corresponding `Location` instance with the same attributes (`name`, `description` and etc) will be created.
+3. If you only have `Site` instances in your existing database, a `Site` LocationType will be created and every root level `LocationType` in your database will have the new `Site` LocationType as their parent. For each legacy `Site` instance, a corresponding `Location` instance with the same attributes (`name`, `description`, `tenant`, `facility`, `asn`, `latitude`, `longitude` and etc) will be created. In addition to that, models (`CircuitTermination`, `Device`, `PowerPanel`, `RackGroup`, `Rack`, `Prefix`, `VLANGroup`, `VLAN`, `Cluster`) with an old `Site` instance assigned without its `location` attribute specified will have its `location` point to the new `Location` of `Site` LocationType. These Models with `location` attribute specified will remain unchanged.
+
+    For Example:
+        A `Site` instance with name **AMS01** is the base site for two top level locations with names **root-01** and **root-02** respectively in the your existing database. During the data migration, `Site` LocationType and a location of this LocationType named **AMS01** with all the information (`asn`, `latitude`, etc) will be created and **root-01** and **root-02** will have this location as their `parent`.
```

Comment:
```suggestion
        We will start with a `Site` instance with name **AMS01** as the base `Site` for two top-level `Location` objects with names **root-01** and **root-02** respectively. During the data migration, a `LocationType` named **Site** will be created, and a `Location` of **Site** `LocationType` named **AMS01** with all the information (`asn`, `latitude`, etc.) from the base `Site` will be created. The `Location` objects named **root-01** and **root-02** will have this **AMS01** `Location` set as their `parent`.
```

Reply:
Any reason not to use quotes instead of bolding for these name strings?

Reply:
To make them stand out.